GOVERNANCE.md

```diff
@@ -25,4 +26,25 @@ Contributors are encouraged to ask for collaborator access once you have had a f
 
 ## Process for releasing a new version of Wikimate
 
-[TODO]
+A PR should be created with all relevant changes to update the repository for the upcoming release.
+(See [#81](https://github.com/hamstar/Wikimate/pull/81) for an example.)
+It should apply the following actions:
+
+1. Change the "Unreleased" heading in the CHANGELOG.md file
```

Comment:
I see, thanks for the clarification. TBH, none appeal much to me, nor appall me, but it took me a while to realize (or remember from the time when I choose "Current development") why.

On Git(Hub) the formal meaning of "release" is to tag the current state of affairs into something fixed, no longer in flux. But to me the very act of putting something online is already "releasing" it, in the practical meaning. That's how I worked with my previous, non-gitted projects; I didn't put anything online until it had enough quality and quantity to be version-able.

So "unreleased" or (worse) "next release" feels like a misnomer because the content under that heading is already _in_ the master branch, is already "out there". I guess I'm still not quite used to GitHub's meaning of "release". To my equally non-native ears my original choice of header sounds like a adequate description of the entries below it, which can remain in a state of flux until tagged (see the number of PRs until WikiFile was ready).

But if you're still feeling awkward about it, then I'm fine with simply sticking to the Keepachangelog standard verbatim. We are, after all, working on GitHub now. :smile: